mesos-simulation/src/main/scala/mesosphere/mesos/simulation/DriverActor.scala

```diff
@@ -45,9 +48,21 @@ class DriverActor(schedulerProps: Props) extends Actor {
 
   private[this] val numberOfOffersPerCycle: Int = 10
 
+  private val randomSeed: Long = 123
+  // use seed for getting more deterministic behavior
+  private[this] val random = new Random(new java.util.Random(randomSeed))
```

Comment:
@gkleiman In order to have more deterministic tests, I'd keep this as suggested by @kolloch.
Is that ok for you?